draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+             | ns-topo  | ietf-ns-topo          | RFCXXXX   |
+             +----------+-----------------------+-----------+
+
+             Table 1: Prefixes and Corresponding YANG Modules
+
+   RFC Editor Note: Please replace XXXX with the RFC number assigned to
+   this document.  Please remove this note.
+
+2.  Modeling Considerations
+
+   An IETF network slice is modeled as network topology defined in
+   [RFC8345], with augmentations.  A new network type "network-slice" is
+   defined in this document.  When a network topology data instance
+   contains the network-slice network type, it represents an instance of
+   an IETF network slice topology.
+
+2.1.  Relationships to Related Topology Models
+
+   There are several related YANG data models that have been defined in
+   IETF.  Some of these are:
+
+   Network Topology Model: Defined in [RFC8345].
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 4]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+   Network Slicing Model: Defined in
+   [I-D.ietf-teas-ietf-network-slice-nbi-yang].
+
+   OTN Slicing: Defined in [I-D.ietf-ccamp-yang-otn-slicing].
+
+   Figure 1 shows the relationships among these models.  The box of
+   dotted lines denotes the model defined in this document.
+
+        +----------+                 +----------+
+        | Network  |                 | Network  |
+        | Slice    |                 | Topology +
+        | NBI YANG >------+          | Model    |
+        | Model    |      |          | RFC 8345 |
+        +----+-----+      |          +-----+----+
+             |            |                |
+             |augments    |references      |augments
+             |            |                |
+        +----^-----+      |          ......^.....
+        | OTN      |      +----------: Network  :
+        | Slicing  | augments        : Slice    :
+        | Model    >-----------------: Topology :
+        |          |                 : Model    :
+        +----------+                 ''''''''''''
+
+                       Figure 1: Model Relationships
+
+2.2.  ACTN for Network Slicing
+
+   Since ACTN topology data models are based on the network topology
+   model defined in [RFC8345], the augmentations defined in this
+   document are effective augmentations to the ACTN topology data
+   models, resulting in making the ACTN framework [RFC8453] and data
+   models [I-D.ietf-teas-actn-yang] capable of slicing networks with the
+   required network characteristics.
+
+3.  Model Applicability
+
+   There are many technologies to achieve network slicing.  The data
+   model defined in this document can be used to configure resource-
+   based network slices, where the resources of a network slice is
+   represented in the form of an abstract network topology, which can
+   then be mapped to a network resource partition (NRP) according to the
+   scenarios defined in [I-D.ietf-teas-ietf-network-slices].
+
```

Comment:
We could add here why a slicing based on topology could be required as described in OTN slicing and putiing on behalf of "OTN" , "transport". 
"Transport slices may be abstracted differently depending on the requirement contained in the configuration provided by the slice customer.
   Whereas the customer requests a transport slice to provide just connectivity between specified endpoints, a transport slice can be represented  as a set of endpoint-to-endpoint links, with each link formed by an end-to-end
 tunnel across the underlying OTN networks.  The resources associated with each link of the slice is reserved and commissioned in the underlying physical network upon the completion of configuring the transport slice and all the links are active.
Alternatively  a transport slice can also be abstracted as an abstract topology when the customer requests the slice to share resources between multiple endpoints and to use the resources on demand.  The abstract topology may consist of virtual nodes and virtual links, and their associated resources are reserved but not commissioned across the underlying transport networks.  
The customer can later commission resources within the slice dynamically using the NBI provided by the service provider.  A transport slice could use abstract topology to connect endpoints with shared resources to optimize the resource utilization, and connections can be activated within the slice as needed.

Reply:
Will incorporate the above to the text.